README.md

```diff
@@ -135,6 +135,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Generate Release Notes Based on Git References](https://github.com/metcalfc/changelog-generator)
 - [Enforce Policies on GitHub Repositories and Commits](https://github.com/talos-systems/conform)
 - [Auto Label Issue Based on Issue Description](https://github.com/Renato66/auto-label)
+- [Expose Git Commit Data As Environment Variables](https://github.com/marketplace/actions/git-commit-data)
```

Comment:
Please link the repository instead of the marketplace :)

Reply:
@jessicalostinspace the link is updated